pkgs/servers/http/apache-modules/mod_scep/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, pkgconfig, mod_ca, apr, aprutil }:
+
+stdenv.mkDerivation rec {
+  pname = "mod_scep";
+  version = "0.2.1";
+
+  src = fetchurl {
+    url = "https://redwax.eu/dist/rs/${pname}-${version}.tar.gz";
+    sha256 = "14l8v6y6kx5dg8avb5ny95qdcgrw40ss80nqrgmw615mk7zcj81f";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs =[mod_ca apr aprutil];
+  inherit(mod_ca) configureFlags installFlags;
```

Comment:
@jtojnar I don't see this often... are you OK with this?

Reply:
Only a mild dislike but, as long as it is communicated in `mod_ca` that the attributes are shared with reverse dependencies, it is the maintainer’s pain.

Reply:
Ok - documented.